apps/explorer/src/app/routes/oracles/components/oracle-markets.tsx

```diff
@@ -61,3 +55,14 @@ export function OracleMarkets({ id }: OracleMarketsProps) {
     </TableRow>
   );
 }
+
+export function getLabel(
+  id: string,
+  m: ExplorerOracleForMarketsMarketFragment | null
+): string {
+  const settlementId =
+    m?.tradableInstrument?.instrument?.product?.dataSourceSpecForSettlementData
+      ?.id || null;
+
+  return id === settlementId ? 'Settlement for' : 'Termination for';
```

Comment:
little note here: we're going to have 3 types of data source specs coming 
* settlement (future, perps)
* termination (futures)
* settlement schedule (perps)

Reply:
Thanks for that. I'll make a ticket to fix this because it looks like it'll end up saying 'Termination for' for perps settlements 🤦